Documentation/project-docs/contributing.md

```diff
@@ -26,6 +26,12 @@ Pull Requests
 * **DO** refer to any relevant issues, and include [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) that automatically close issues when the PR is merged.
 * **DO** tag any users that should know about and/or review the change.
 * **DO** ensure each commit successfully builds.  The entire PR must pass all tests in the Continuous Integration (CI) system before it'll be merged.
-* **DO** address PR feedback in an additional commit(s) rather than ammending the existing commits, and only rebase/squash them when necessary.  This makes it easier for reviewers to track changes. If necessary, squashing should be handled by the merger using the ["squash and merge"](https://github.com/blog/2141-squash-your-commits) feature, and should only be done by the contributor upon request.
+* **DO** address PR feedback in an additional commit(s) rather than amending the existing commits, and only rebase/squash them when necessary.  This makes it easier for reviewers to track changes.
 * **DO NOT** fix merge conflicts using a merge commit. Prefer `git rebase`.
 * **DO NOT** mix independent, unrelated changes in one PR. Separate real product/test code changes from larger code formatting/dead code removal changes. Separate unrelated fixes into separate PRs, especially if they are in different assemblies.
+
+Merging Pull Requests (for contributors with write access)
+----------------------------------------------------------
+
+* **DO** use ["Squash and Merge"](https://github.com/blog/2141-squash-your-commits) by default for individual contributions unless requested otherwise in the PR. Do so, even if the PR contains only one commit. It creates a simpler history than "Create a Merge Commit".
```

Comment:
"unless requested otherwise in the PR"... requested by who?  I'd like to see some examples included here of cases where we'd still use Create a Merge Commit.